sdk/trace/src/main/java/io/opentelemetry/sdk/trace/export/SimpleSpanProcessorBuilder.java

```diff
@@ -29,12 +34,26 @@ public SimpleSpanProcessorBuilder setExportUnsampledSpans(boolean exportUnsample
     return this;
   }
 
+  /**
```

Comment:
No `setInternTelemetryVersion` here? Is the though to add it only when its needed?

Reply:
Yeah I figured so, there's no LEGACY here so it would be a literal no-op method.

Reply:
Maybe put keep a TODO in these places to remind us to add it when needed?

```
// TODO: add `setInternalTelemetryVersion when we support more than one version
```